src/hooks/useLogin.tsx

```diff
@@ -1,7 +1,8 @@
 import { FormEventHandler, useState } from "react"
 
 export default function useLogin() {
-  const [loggedIn, setLoggedIn] = useState(false)
+  const currentState = Boolean(localStorage.getItem("accessToken"))
```

Comment:
This may cause some unexpected issues. React runs the component body and every hook attached to it on every re-render. I can't see it breaking now, but when setting initial values you should use onMounted hooks or similar.

Reply:
This works as expected. My bad, sorry for confusion